SampleWebApplicationVB/Web.Release.config

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
```

Comment:
As this runs locally, we can remove the `Web.Release.config` and `Web.Debug.config` transformations.

Reply:
Done